styles.css

```diff
@@ -289,11 +280,85 @@ textarea:focus {
 /* --------------- m o b i l e ----------------------- */
 
 @media only screen and (max-width: 767px) {
+  header {
+    margin-left: 5%;
+    display: flex;
+    justify-content: space-between;
+    background-color: white;
+    width: 100%;
+    padding-top: 5%;
+  }
+
   .logo h3 {
-    font-weight: 600px;
+    font-weight: 600;
     font-size: 20px;
   }
 
+  /* dropdown menu start */
+  .container {
+    display: inline-block;
+    cursor: pointer;
+    align-items: center;
+  }
+
+  .bar1,
+  .bar2,
+  .bar3 {
+    width: 35px;
+    height: 5px;
+    background-color: #333;
+    margin: 6px 0;
+    transition: 0.4s;
+  }
+
+  .change .bar1 {
+    transform: translate(0, 11px) rotate(-45deg);
+    background-color: #67798e;
+  }
+
+  .change .bar2 {
+    opacity: 0;
+  }
+
+  .change .bar3 {
+    transform: translate(0, -11px) rotate(45deg);
+    background-color: #67798e;
+  }
+
+  .accordion {
+    color: #444;
+    cursor: pointer;
+    width: 30%;
+    border: none;
+    outline: none;
+    font-size: 15px;
+    transition: 0.4s;
+  }
+
+  #panel {
+    padding: 0 18px;
+    display: none;
+    background-color: white;
+    overflow: hidden;
+  }
+
+  #panel a {
+    display: flex;
+    text-decoration: none;
+    padding: 10px 0;
+    font-size: 1.5rem;
+    font-weight: 700;
+    color: #3a4a42;
+    left: 5%;
+    flex: 100%;
+  }
```

Comment:
- [ ] Kindly make the `font-size` of the links in the mobile menu `2rem`. That way, your mobile menu will look identical to that of the Figma template that you chose. 👍

Reply:
I've done it